_data/footer.yml

```diff
@@ -1,6 +1,8 @@
 title: FEEL FREE TO CONTACT US
 email: contact@implex.dev
 subtitle: Looking forward to connecting with you
-href: https://www.linkedin.com/company/implex-dev/about/
+linkedinHref: https://www.linkedin.com/company/implex-dev/about/
+youtubeHref: https://www.youtube.com/@implex6589
 creator: © 2020 Implex. All rights reserved.
-icon: linkedin.svg
+linkedinIcon: linkedin.svg
+youtubeIcon: youtube.svg
```

Comment:
I would recommend you to have an array (list) of icons: 
  - icon: ...
    href: ...
 - icon: ...
    href: ...

In this case 
1) no copy-paste of `<div class="icon">` block
2) easy-peasy to add one more icon, let's say facebook